style.css

```diff
@@ -0,0 +1,832 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+.navbar {
+  position: sticky;
+  padding: 5px;
+  top: 0;
+}
+
+a {
+  color: black;
+}
+
+.navbar a {
+  text-decoration: none;
+  font-family: 'Lato', sans-serif;
+}
+
+.material-symbols-outlined {
+  font-variation-settings:
+    'FILL' 0,
+    'wght' 700,
+    'GRAD' 0,
+    'opsz' 48;
+  font-size: 45px;
+}
+
+#bg {
+  background-image: url(Images/graybg.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#bg-about {
+  background-color: #f4f4f4;
+  background-image: url(Images/graybg-ABOUT.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  column-gap: 200px;
+  margin-inline: 5%;
+
+  /* height: 500px;    */
+}
+
+.main p {
+  background-color: rgb(240, 240, 240);
+  width: 90%;
+  border: 3px solid white;
+  font-family: 'Lato', sans-serif;
+  line-height: 30px;
+  padding: 10px;
+}
+
+.about {
+  justify-content: center;
+}
+
+.about #sup_text {
+  background-color: white;
+  width: 90%;
+  margin-inline: 2.5%;
+  border: 1px solid rgb(177, 177, 177);
+  font-family: 'Lato', sans-serif;
+  padding: 10px;
+  margin-top: 50px;
+  line-height: 30px;
+}
+
+.about #suptext2 {
+  width: 60%;
+  margin-inline: 20%;
+}
+
+h3 {
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+  font-weight: 100;
+}
+
+.about h3 {
+  align-self: center;
+}
+
+.about #sup_text_desk {
+  display: none;
+  background-color: white;
+  width: 90%;
+  margin-inline: 2.5%;
+  border: 1px solid rgb(177, 177, 177);
+  font-family: 'Lato', sans-serif;
+  padding: 10px;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+.about p {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  margin-bottom: 30px;
+}
+
+h2,
+h4 {
+  color: #424242;
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+}
+
+.header h4 {
+  margin-bottom: 40px;
+}
+
+.header h2 {
+  font-weight: bolder;
+  font-size: 25px;
+}
+
+.main h1 {
+  background-image: url(Images/h1bg.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 50px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-family: 'CocoGoose', sans-serif;
+  line-height: 45px;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+.about h1 {
+  color: #ec5242;
+  font-size: 50px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-family: 'CocoGoose', sans-serif;
+  line-height: 48px;
+  margin-top: 20px;
+  margin-bottom: 0;
+  text-align: center;
+}
+
+#program {
+  background-image: url(Images/Abstract-Patterns-High-Quality-Gray-Background.webp);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding-bottom: 80px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- Let's reduce the paddings to match the design as you space is too much.

*main design*
![spx](https://user-images.githubusercontent.com/59397159/181862804-badc5ef8-fc5c-405f-8199-b4efc272d600.png)

*your design*
![space](https://user-images.githubusercontent.com/59397159/181862805-19c8c964-8119-4c08-b374-56eac9d922d4.png)